src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -36,7 +36,7 @@ const EmbedSnippetPanel = ({ startTime }: EmbedSnippetPanelProps) => {
     const [showPreview, setShowPreview] = React.useState(false);
 
     const url = `https://simularium.allencell.org/embed?trajFileName=${trajectory}&t=${startTime}`;
-    const embedSnippet = `<iframe height="${height}" width="${width}" src="${url}" title="Simularium" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>`;
+    const embedSnippet = `<iframe width="${width}" height="${height}" src="${url}" title="Simularium" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>`;
```

Comment:
to see the small controls during development the url needs to be changed to this:
`http://localhost:9001/embed?trajFileName=${trajectory}&t=${startTime}`

once this is merge the line that is committed should show the small controls

Reply:
you could fix this by using `location.host`